rectools/dataset/features.py

```diff
@@ -450,5 +450,21 @@ def __len__(self) -> int:
         """Return number of objects."""
         return self.values.shape[0]
 
+    def get_cat_features(self) -> "SparseFeatures":
+        """Return `SparseFeatures` only with categorical features.""" ""
+        cat_features_names_list: tp.List[SparseFeatureName] = []
```

Comment:
You really don't need it.
We can make this piece of code more clean. Always try to write as clean as possible and drop all unnecessary complications. This will make your code much easier to read and maintain.